Circle.cpp

```diff
@@ -1,14 +1,42 @@
 #ifndef Circle_cpp
 #define Circle_cpp
+
+#define _USE_MATH_DEFINES
+
 #include <iostream>
-#include "Draw.cpp"
+#include <math.h>
+
+#include "Shape.cpp"
+#include "Area.cpp"
+#include "Input.cpp"
+
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape, public Input
 {
-public:
-    void drawShape() override
-    {
-        cout << "circle" << endl;
-    }
+
+    private:
+        double radius = input();
```

Comment:
Getting user input upon object creation is not a good practice. Since it couples creation of the object with user input.